src/Components/QuestionComponentContainer/QuestionComponentContainer.css

```diff
@@ -1,3 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400;700&display=swap');
+
+/* h1,
+h3 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #b85a27;
+}
+
+h2 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #293457;
+}
+
+h4,
+h5,
+h6 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #293457;
+}
+*/
```

Comment:
What do you think about removing this comment?

Reply:
Great catch! I've removed the comment. Also removed the font import as that is being handled in another ticket by Marianna.